torch/_dynamo/utils.py

```diff
@@ -978,6 +980,8 @@ def us_to_ms(field):
         "remote_fx_graph_cache_put_time_ms": us_to_ms(
             "remote_fx_graph_cache_put_time_us"
         ),
+        "cuda_version": torch.version.cuda,
```

Comment:
nit: See comments above. Eventually we'd plan to remove most of these fields. Mind moving this up under "inductor_config" so there's no future confusion?

Reply:
Done.